lte/gateway/c/sctpd/util.cpp

```diff
@@ -134,7 +134,10 @@ int convert_addrs(const InitReq *req, struct sockaddr **addrs, int *num_addrs)
     ipv4_addr = (struct sockaddr_in *) &(*addrs)[i];
 
     ipv4_addr->sin_family = AF_INET;
-    ipv4_addr->sin_port = htons(req->port());
+    if (req->port())
```

Comment:
Is the assumption here that an Init Req can come without port information?

Reply:
Yes, added just to avoid empty init req structure, but can be avoided as in the base magma code.